### ISSUE TITLE
about the loss

### ISSUE THREAD

**Issue by xuetingdai** [tianzhi0549/FCOS#96]
when train ，the loss = L(cls)+L(reg)+L(centerness) ?
i don't understand  when we use the centerloss.

**Comment by chhshen** [comment]
Pls read the paper

**Comment by xuetingdai** [comment]
@chhshen the paper showed that loss is  L(cls)+L(reg) in train,
the centerness is only used in testing ?
i read the paper,but i can't understand this

**Comment by tianzhi0549** [comment]
Center-ness is introduced in Sec. 3.3.
